WordPress/Classes/ViewRelated/Activity/ActivityListViewModel.swift

```diff
@@ -168,6 +233,16 @@ class ActivityListViewModel: Observable {
         return firstYear != currentYear || secondYear != currentYear
     }
 
+    private func applyingNewFilter(after: Date? = nil, before: Date? = nil, group: [ActivityGroup]) -> Bool {
+        let isSameGroup = group.count == self.selectedGroups.count && self.selectedGroups.elementsEqual(group, by: { $0.key == $1.key })
+
+        return applyingDateFilter(after: after, before: before) || !isSameGroup
+    }
+
+    private func applyingDateFilter(after: Date? = nil, before: Date? = nil) -> Bool {
```

Comment:
```suggestion
    private func isApplyingDateFilter(after: Date? = nil, before: Date? = nil) -> Bool {
```

Reply:
Done in 6f4fb96128a48d3948cf96d3b5fc1cdb91a8d36a